SpatialGDK/Source/SpatialGDK/Private/Utils/OpUtils.cpp

```diff
@@ -22,6 +22,22 @@ void FindFirstOpOfType(const TArray<Worker_OpList*>& InOpLists, const Worker_OpT
 	}
 }
 
+void FindAllOpsOfType(const TArray<Worker_OpList*>& InOpLists, const Worker_OpType InOpType, TArray<Worker_Op*>& OutFoundOps)
```

Comment:
uber nit: InOpType doesn't match function signature name

Reply:
The other functions in this file follow the same pattern, should I change all of them?